pkg/storage/segments/store.go

```diff
@@ -239,6 +243,7 @@ func makeRemotePointer(nodes []*pb.Node, rs eestream.RedundancyStrategy, pieceID
 		remotePieces = append(remotePieces, &pb.RemotePiece{
 			PieceNum: int32(i),
 			NodeId:   nodes[i].Id,
+			Hash:     hashes[i],
```

Comment:
Where does the uplink validates the hash before including it in the pointer? Or is it going to be a separate PR?

Reply:
It was done in previous PR - https://github.com/storj/storj/blob/81408a3c9e0637024d4c544252071b41492dc88a/pkg/piecestore/psclient/readerwriter.go#L74

Reply:
Cool. I missed it.